### ISSUE TITLE
Children without parents

### ISSUE THREAD

**Issue by Cr4psy** [northvolt/uataglist#43]
Certain tags can be generated without having existing parents if the excel file is wrong.
It would be useful to check that the parents exist.

**Comment by lgrip** [comment]
Agreed, I have thought of adding this check. Need to decide what to do with these tags (probably just not include them and note that they have been skipped)
